lib/vsc/mympirun/rm/sched.py

```diff
@@ -222,37 +227,23 @@ def is_large(self):
         """Determine if this is a large job or not"""
 
         res = ((len(self.nodes) > self.RSH_LARGE_LIMIT) and
-               (self.ppn == self.cores_per_node))
+               (any(c == self.cores_per_node for c in self.ppn_dict.values())))
         self.log.debug("is_large returns %s", res)
         return res
 
-    def set_mpiprocesspernode(self):
-        """set mpiprocesspernode """
+    def set_multiplier(self):
+        """set multiplier """
 
         # get the working mode from options
         hybrid = getattr(self.options, 'hybrid', None)
         double = getattr(self.options, 'double', False)
 
-        # set the multiplier
         if hybrid:
-            multi = hybrid
+            self.multiplier = hybrid
         elif double:
-            multi = 2
+            self.multiplier = 2
```

Comment:
this is intended, specifying `--hybrid` and `--double` means `use hybrid as multiplier`